app/classifier/task-nav.jsx

```diff
@@ -122,78 +91,34 @@ class TaskNav extends React.Component {
-              onBlur={this.warningToggleOff}
-            >
-              <Translate content="classifier.back" />
-            </button>}
-          {(!nextTaskKey && this.props.workflow.configuration.hide_classification_summaries && this.props.project && !disableTalk && !completed) &&
-            <Link
-              onClick={this.completeClassification}
-              to={`/projects/${this.props.project.slug}/talk/subjects/${this.props.subject.id}`}
-              className="talk standard-button"
-              style={waitingForAnswer ? disabledStyle : {}}
-            >
-              <Translate content="classifier.doneAndTalk" />
-            </Link>}
-          {(nextTaskKey && this.props.annotation && !this.props.annotation.shortcut) ?
-            <button
-              type="button"
-              className="continue major-button"
-              disabled={waitingForAnswer}
-              onClick={this.addAnnotationForTask.bind(this, nextTaskKey)}
-            >
-              <Translate content="classifier.next" />
-            </button> : !completed ?
-              <button
-                type="button"
-                className="continue major-button"
-                disabled={waitingForAnswer}
-                onClick={this.completeClassification}
-              >
-                {this.props.demoMode && <i className="fa fa-trash fa-fw" />}
-                {this.props.classification.gold_standard && <i className="fa fa-star fa-fw" />}
-                {' '}<Translate content="classifier.done" />
-              </button> :
-              null
-          }
-          {completed &&
-            <Link
-              onClick={this.props.nextSubject}
-              to={`/projects/${this.props.project.slug}/talk/subjects/${this.props.subject.id}`}
-              className="talk standard-button"
-            >
-              <Translate content="classifier.talk" />
-            </Link>}
-          {completed &&
-            <button
-              autoFocus={this.props.autoFocus}
-              className="continue major-button"
-              onClick={this.props.nextSubject}
-            >
-              <Translate content="classifier.next" />
-            </button>}
+          <TaskNavButtons
+            addAnnotationForTask={this.addAnnotationForTask.bind(this, nextTaskKey)}
+            areAnnotationsNotPersisted={!this.props.workflow.configuration.persist_annotations}
+            autoFocus={this.props.autoFocus}
+            classification={this.props.classification}
+            completeClassification={this.completeClassification}
+            completed={completed}
+            demoMode={this.props.demoMode}
+            destroyCurrentAnnotation={this.destroyCurrentAnnotation}
+            nextSubject={this.props.nextSubject}
+            project={this.props.project}
+            showBackButton={visibleTasks.length > 1 && !completed && (this.props.classification.annotations.indexOf(this.props.annotation) !== 0)}
```

Comment:
I think `this.props.classification.annotations` should be `this.props.annotations`

Reply:
Good catch. I likely missed this from the PR that changed that prop.

Reply:
Should fix the Back button showing on the first task.